config/boards/orangepi5.wip

```diff
@@ -0,0 +1,15 @@
+# Rockchip RK3588S octa core 4/8/16GB RAM SoC NVMe USB3 USB-C GbE
+BOARD_NAME="Orange Pi 5"
+BOARDFAMILY="rockchip-rk3588" # shared family with rock-5b # separate family: BOARDFAMILY="orangepi-rk3588"
+BOOTCONFIG="orangepi_5-rk3588s_defconfig"
+BOOT_SOC="rk3588" # would be `3588s` if left to BOOTCONFIG parsing
```

Comment:
Why rk3588? I think rk3588s would be better

Reply:
`BOARDFAMILY="rockchip-rk3588"`: we wanna re-use the existing family, so no `s` there.

`BOOT_SOC="rk3588"`: to avoid adding an `if` branch to rockchip64_common just to account for this one. Everything else is the same, blobs etc, so no reason to have a separate one until we do.

Reply:
Understood bro